### PR TITLE
search by encounter filters patient level document

### PR DESCRIPTION
**Pull request by ramya1096**
Description
R4 DocumentReference search by encounter should not return patient level documents

Screenshots

Screenshot 2021-07-16 at 3 08 13 PM

Screenshot 2021-07-16 at 10 51 05 AM

PR Checklist
----
- [X] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/foundation/documents/document-reference.md [hunk 1]
```diff
@@ -81,6 +84,20 @@ _Implementation Notes_
 
 <%= disclaimer %>
 
+### Example: search by encounter filters patients level documents
+
+#### Request
+
+    GET https://fhir-ehr.cerner.com/r4/2c400054-42d8-4e74-87b7-80b5bd5fde9f/DocumentReference?patient=823932&encounter=863887
```
Comment (ashu-bdvt) [inline content/millennium/r4/foundation/documents/document-reference.md:91]: Replace the dev tenant id with c1941 tenant "ec2458f2-1e24-41c8-b71b-0e701af7583d" at all places

**Review by ashu-bdvt — CHANGES_REQUESTED** [review]
Update request to have sandbox tenant id

File: content/millennium/r4/foundation/documents/document-reference.md (hunk 1 above)
Comment (ramya1096) [reply content/millennium/r4/foundation/documents/document-reference.md]: changes addressed

**Comment by UjwalaBheema** [comment]
> Update request to have sandbox tenant id

+1

**Review by UjwalaBheema — CHANGES_REQUESTED** [review]
Update screenshots with c1941 changes

**Review by UjwalaBheema — APPROVED** [review]

**Review by ashu-bdvt — APPROVED** [review]

**Comment by ramya1096** [comment]
search by encounter filters patient level document

**Comment by zxeleven** [comment]
Verified live:

Screen Shot 2021-08-05 at 4 33 15 PM
Screen Shot 2021-08-05 at 4 34 00 PM

**Pull request merged**